### PR TITLE
Format the durationInhours value

### PR DESCRIPTION
**Pull request by khansaad**
# Format the `DurationInHours` value to two decimal places

## Description

This PR will do the following changes - 
- update the code to format the `DurationInHours` value to restrict it to 2 decimal places.

### Type of change

- [x] Refactoring Code
- [ ] New feature

## How has this been tested?

- Using kruize-demo scripts. 
Image: `quay.io/khansaad/autotune_operator:round-off`
- Manually verified the value updated to 2 decimal places. 

**Test Configuration**
* Kubernetes clusters tested on: minikube

## Checklist :dart:

- [X] Followed coding guidelines
- [X] Comments added
- [x] Dependent changes merged
- [ ] Documentation updated
- [ ] Tests added or updated

## Additional information
-

### CODE REVIEW

**Comment by khansaad** [comment]
Test results attached:
https://ci.app-svc-perf.corp.redhat.com/job/ExternalTeams/job/Autotune/job/kruize_test_pull_request/104/

@chandrams Do we need to add new test for this ?

**Comment by chandrams** [comment]
@khansaad - No need of a new test, you can add a check in the validation logic for recommendations to check for expected precision for duration in hours.

File: src/main/java/com/autotune/analyzer/recommendations/term/Terms.java [hunk 1]
```diff
@@ -146,6 +145,8 @@ public static void setDurationBasedOnTerm(ContainerData containerDataKruizeObjec
             durationSummation = maxDurationInMinutes;
         }
         double durationSummationInHours = durationSummation / KruizeConstants.TimeConv.NO_OF_MINUTES_PER_HOUR;
+        durationSummationInHours = Double.parseDouble(String.format("%.2f", durationSummationInHours));
+        LOGGER.debug("durationSummationInHours = {}", durationSummationInHours);
```
Comment (dinogun) [inline src/main/java/com/autotune/analyzer/recommendations/term/Terms.java:149]: Can we remove this LOGGER, this is not useful as the info is available in the JSON anyway

Comment (khansaad) [reply src/main/java/com/autotune/analyzer/recommendations/term/Terms.java]: Done!

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**